applications/luci-app-adblock-fast/htdocs/luci-static/resources/adblock-fast/status.js

```diff
@@ -198,7 +198,7 @@ var status = baseclass.extend({
 							"Use of external dnsmasq config file detected, please set '%s' option to '%s'"
 						).format("dns", "dnsmasq.conf"),
 						warningMissingRecommendedPackages: _(
-							"Some recommended packages are missing"
+							"Missing recommended package: '%s'"
```

Comment:
I must've missed this - is this missing a `.format()`?

Reply:
It isn't missing the `.format()`, the entire text label is assembled in https://github.com/openwrt/luci/blob/ac1043cdc6a24485a052136b5be1598a21152a52/applications/luci-app-adblock-fast/htdocs/luci-static/resources/adblock-fast/status.js#L318-L319 where the `.format()` is present.